tests/devices/test_default_qubit.py

```diff
@@ -771,289 +755,6 @@ def circuit(x):
-                                                          ])
-    def test_inverse_circuit_with_parameters_expectation(self, qubit_device_1_wire, tol, name, expected_output, phi):
-        @qml.qnode(qubit_device_1_wire)
-        def circuit():
-            qml.RX(phi, wires=0)
-            qml.RX(phi, wires=0).inv()
-            return qml.expval(qml.PauliZ(0).inv())
-
-        assert np.isclose(circuit(), expected_output, atol=tol, rtol=0)
-
-    # This test is ran against the state |0> with one Z expval
-    @pytest.mark.parametrize("name,expected_output", [
-        ("PauliX", -1),
-        ("PauliY", -1),
-        ("PauliZ", 1),
-        ("Hadamard", 0),
-    ])
-    def test_supported_gate_single_wire_no_parameters(self, qubit_device_1_wire, tol, name, expected_output):
-        """Tests supported gates that act on a single wire that are not parameterized"""
-
-        op = getattr(qml.ops, name)
-
-        assert qubit_device_1_wire.supports_operation(name)
-
-        @qml.qnode(qubit_device_1_wire)
-        def circuit():
-            op(wires=0)
-            return qml.expval(qml.PauliZ(0))
-
-        assert np.isclose(circuit(), expected_output, atol=tol, rtol=0)
-
-    # This test is ran against the state |Phi+> with two Z expvals
-    @pytest.mark.parametrize("name,expected_output", [
-        ("CNOT", [-1/2, 1]),
-        ("SWAP", [-1/2, -1/2]),
-        ("CZ", [-1/2, -1/2]),
-    ])
-    def test_supported_gate_two_wires_no_parameters(self, qubit_device_2_wires, tol, name, expected_output):
-        """Tests supported gates that act on two wires that are not parameterized"""
-
-        op = getattr(qml.ops, name)
-
-        assert qubit_device_2_wires.supports_operation(name)
-
-        @qml.qnode(qubit_device_2_wires)
-        def circuit():
-            qml.QubitStateVector(np.array([1/2, 0, 0, math.sqrt(3)/2]), wires=[0, 1])
-            op(wires=[0, 1])
-            return qml.expval(qml.PauliZ(0)), qml.expval(qml.PauliZ(1))
-
-        assert np.allclose(circuit(), expected_output, atol=tol, rtol=0)
-
-    @pytest.mark.parametrize("name,expected_output", [
-        ("CSWAP", [-1, -1, 1]),
-    ])
-    def test_supported_gate_three_wires_no_parameters(self, qubit_device_3_wires, tol, name, expected_output):
-        """Tests supported gates that act on three wires that are not parameterized"""
-
-        op = getattr(qml.ops, name)
-
-        assert qubit_device_3_wires.supports_operation(name)
-
-        @qml.qnode(qubit_device_3_wires)
-        def circuit():
-            qml.BasisState(np.array([1, 0, 1]), wires=[0, 1, 2])
-            op(wires=[0, 1, 2])
-            return qml.expval(qml.PauliZ(0)), qml.expval(qml.PauliZ(1)), qml.expval(qml.PauliZ(2))
-
-        assert np.allclose(circuit(), expected_output, atol=tol, rtol=0)
-
-    # This test is ran with two Z expvals
-    @pytest.mark.parametrize("name,par,expected_output", [
```

Comment:
took the liberty of abandoning this test case (quite many other basis state tests)